README.md

```diff
@@ -85,63 +97,44 @@ AsmodeeNet.init({
-* **base_is_host** *string* Optional, URL of the AN Identity Server. By default, it's https://account.asmodee.net but you can set https://account.staging.asmodee.net for your test by ex.
-* **base_is_path** *string* Optional. This should be used, in futur, to use an other version of the IS oAuth API. default /main/v2/oauth.
-* **display** *string* Optional. Defines the way the OAuth flow should be handled. Possible values are `popup`, `touch` and `page`:
- * `popup` opens a popup which dimensions can be set up with the `signIn()` method (see below);
- * `touch` keeps the user in the same window or tab and provides a clean layout for the login page, suitable for mobile displays (phones or tablets);
- * `page` keeps the user in the same window or tab and provides a standard layout for the login page.
-* **display_options** *object* Optional. Defines some display options for signin/signup pages of IS. Usable with `display` at `popup` or `touch`. Available options are:
-    * `noheader` **boolean** Default false. If true, navbar and header of IS pages don't be displayed
-    * `nofooter` **boolean** Default false. If true, footer of IS pages don't be displayed
-    * `lnk2bt`   **boolean** Default false. If true, Some link of formules will be transformed in button.
-    * `leglnk`   **boolean** Default true. If false, some legals link don't be displayed
-  If you set `display` to `touch`, and don't set this options `display_options`, their options are set to:
-    * `noheader` to true
-    * `nofooter` to true
-    * `lnk2bt`   to true
-    * `leglnk`   to false
-* **callback_signin_success** *callback* Optional. The function to call after a successful sign-in. Default is `console.log`.
-* **callback_signin_error** *callback* Optional. The function to call after an unsuccessful sign-in. Default is `console.error`.
-
-##### Discover
-
-After the init phase, you could call the discover method to use to discovery capacity of OpenID Connect to auto-configure other parts of the library.
+You can override the other default options by giving additional parameters to the `init()` function.
+
+Parameters:
+* **client_id** *string* Mandatory. The app client ID.
+* **redirect_uri** *string* Mandatory. The redirection URI.
+* **cancel_uri** *string* Optional. Used only if `display` is set to `touch` or `popup` and `display_options.lnk2bt` is true. In this case, if `cancel_uri` is not set, it takes the value of redirect_uri. This value will be used by the Identity Server for the cancel button. See the `display` and `display_options` parameters for more information.
+* **logout_redirect_uri** *string* Optional.
+* **callback_post_logout_redirect** *callback* Optional. This URI will be called after a logout.
+* **response_type** *string* Optional, default is `'token id_token'`. Values can be: `token`, `id_token` or both as space-separated words. We strongly recommend that you use the default value.
```

Comment:
It's not from you change, but i remark that it's missing `code`as possible value. So the phrase should be:

Optional, default is `'token id_token'`. Values can be: `code`, `token`, `id_token` or any space-separated combination of three.